Server/mods/deathmatch/logic/CPlayer.h

```diff
@@ -203,8 +203,8 @@ class CPlayer final : public CPed, public CClient
     bool IsCursorShowing() { return m_bCursorShowing; }
     void SetCursorShowing(bool bCursorShowing) { m_bCursorShowing = bCursorShowing; }
 
-    char* GetNametagText() { return m_szNametagText; }
-    void  SetNametagText(const char* szText);
+    std::string_view GetNametagText() { return m_nametagText; }
```

Comment:
We need to be very careful with things like this, as `std::string_view` may not be null-terminated. If other code did something like `if(strlen(pPlayer->GetNametagText()) > 42)`, it could fail.

Reply:
Well, there's no implicit conversion to `const char*`. Also I had to refactor a few functions using this method, but thats it, nothing else uses it.

Reply:
Resolved?